### ISSUE TITLE
export with --convert-to-jpeg fails for live photos in HEIC format

### ISSUE THREAD

**Issue by garz75** [RhetTbull/osxphotos#235]
with 0.35.1 
Steps to reproduce : 

- create an album 'osxphotos' with one live photo in HEIC format in it 
- run command : 
```
$ osxphotos export . --update --album "osxphotos" --convert-to-jpeg --jpeg-quality 0.95 --directory "{folder_album,NoAlbum}"  --person-keyword --keyword-template "{label}"  --exiftool  --filename "{created.strftime,%Y-%m-%d-%H%M%S}"
Using last opened Photos library: /Users/fgarzon/Pictures/Bibliothèque Photos.photoslibrary
Exporting 1 photo to ....

Traceback (most recent call last):
  File "/usr/local/bin/osxphotos", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/decorators.py", line 33, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/osxphotos/__main__.py", line 1660, in export
    results = export_photo(
  File "/usr/local/lib/python3.8/site-packages/osxphotos/__main__.py", line 2281, in export_photo
    export_results = photo.export2(
  File "/usr/local/lib/python3.8/site-packages/osxphotos/photoinfo/_photoinfo_export.py", line 585, in export2
    results = self._export_photo(
  File "/usr/local/lib/python3.8/site-packages/osxphotos/photoinfo/_photoinfo_export.py", line 963, in _export_photo
    converted_stat = fileutil.file_sig(dest_str)
  File "/usr/local/lib/python3.8/site-packages/osxphotos/fileutil.py", line 171, in file_sig
    return cls._sig(os.stat(f1))
FileNotFoundError: [Errno 2] No such file or directory: 'osxphotos/2019-08-05-174931.mov'
$ > ll osxphotos/
total 8704
-rwxrwxrwx  1 fgarzon  staff  4436826 Oct 15 08:35 2019-08-05-174931.jpeg
```

Expected result : the .mov file should beexported, alongside the .jpeg version of the HEIC photo.

Note that the same command succeeds with the --skip-live option : 
```
$ osxphotos export . --update --album "osxphotos" --convert-to-jpeg --jpeg-quality 0.95 --directory "{folder_album,NoAlbum}"  --person-keyword --keyword-template "{label}"  --exiftool --skip-live  --filename "{created.strftime,%Y-%m-%d-%H%M%S}"
Using last opened Photos library: /Users/fgarzon/Pictures/Bibliothèque Photos.photoslibrary
Exporting 1 photo to ....
  [####################################]  100%
Exported: 1 photo, updated: 0 photos, skipped: 1 photo, updated EXIF data: 1 photo
Elapsed time: 2.356 seconds
$ ll osxphotos/
total 8704
-rwxrwxrwx  1 fgarzon  staff  4440368 Oct 15 08:40 2019-08-05-174931.jpeg
```

**Comment by RhetTbull** [comment]
@garz75 thanks for the comprehensive bug report.  I think I've fixed this in [v0.35.3](https://github.com/RhetTbull/osxphotos/releases/tag/v0.35.3).  I'll keep this issue open until I have time to add tests.

**Comment by garz75** [comment]
It looks like indeed that this fixed the problem. Thanks !
